packages/terra-form-select/src/native-select/NativeSelect.jsx

```diff
@@ -0,0 +1,272 @@
+import React, {
+  useState,
+  useRef,
+} from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import classNamesBind from 'classnames/bind';
+import { injectIntl, intlShape } from 'react-intl';
+import ThemeContext from 'terra-theme-context';
+import {
+  defaultPlaceholderValue,
+  isCurrentPlaceholder,
+  isValuePresent,
+  getDisplay,
+  getFirstValue,
+  getOptGroupKey,
+  getOptGroupKeyIndex,
+} from './_NativeUtils';
+import {
+  optionPropType,
+  optGroupPropType,
+} from './_NativePropTypes';
+
+import styles from './NativeSelect.module.scss';
+
+const cx = classNamesBind.bind(styles);
+
+const propTypes = {
+  /**
+   * The string containing ids for elements to describe the select.
+   */
+  ariaDescribedBy: PropTypes.string,
+  /**
+   * The aria label string value for the select.
+   */
+  ariaLabel: PropTypes.string.isRequired,
+  /**
+   * Additional attributes to spread directly onto the html select node.
+   * This prop is provided as standard customProps are spread onto the outer containing element which is not the html select in this case.
+   */
+  // eslint-disable-next-line react/forbid-prop-types
+  attrs: PropTypes.object,
+  /**
+   * The default value of the select. Can be a string, or number.
+   */
+  defaultValue: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
+  /**
+   * Whether the input is disabled.
+   */
+  disabled: PropTypes.bool,
+  /**
+   * The Select identifier to be applied to the html select node.
+   */
+  id: PropTypes.string,
+  /**
+   * @private
+   * The intl object to be injected for translations.
+   */
+  intl: intlShape.isRequired,
+  /**
+   * Whether the select input should use the filter style display, forcing a value to be selected.
```

Comment:
@dkasper-was-taken How does this sound to update the prop description? 
```suggestion
   * Whether the select input should use the filter style display, forcing a value to always be selected. It also removes the Terra placeholder and removes the ability for user to clear the value, returning the select to browser-native behavior.
```

Reply:
Updated.